float64 final_cost_penalty
float64 instant_velocity_cost_penalty
float64 waypoint_cost_penalty
float64[] waypoint_timings
float64[] waypoints
float64[] starts
float64[] goals